osu.Game/Screens/Select/BeatmapInfoWedge.cs

```diff
@@ -418,10 +418,18 @@ public InfoLabel(BeatmapStatistic statistic)
                                     Anchor = Anchor.Centre,
                                     Origin = Anchor.Centre,
                                     RelativeSizeAxes = Axes.Both,
-                                    Scale = new Vector2(0.8f),
                                     Colour = Color4Extensions.FromHex(@"f7dd55"),
-                                    Icon = statistic.Icon,
+                                    Icon = FontAwesome.Regular.Circle,
```

Comment:
Isn't this only specifically for osu! rulesets? E.g. rush! now looks like:
![image](https://user-images.githubusercontent.com/1329837/92109615-4c9fdd00-ee24-11ea-9c42-a63d038508ba.png)

Reply:
Well that highly depends on expectation. Should the circle be always present or something optional? Probably need to figure that out and check with @arflyte.

Reply:
I've adjusted the legacy icons to display inside the circle but also be sized correctly. Seems okay for now?

![image](https://user-images.githubusercontent.com/191335/92197809-c6c87400-eead-11ea-8282-2abda7fca1c7.png)

Reply:
Some further tweaks to match flyte's originals better (except for circle thickness):

![image](https://user-images.githubusercontent.com/191335/92198434-5884b100-eeaf-11ea-9926-cd6e4b311c79.png)